client/landing/stepper/declarative-flow/write.ts

```diff
@@ -0,0 +1,141 @@
+import { useLocale } from '@automattic/i18n-utils';
+import { useFlowProgress, WRITE_FLOW } from '@automattic/onboarding';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { translate } from 'i18n-calypso';
+import { useEffect } from 'react';
+import { recordFullStoryEvent } from 'calypso/lib/analytics/fullstory';
+import wpcom from 'calypso/lib/wp';
+import { useSiteSlug } from '../hooks/use-site-slug';
+import { USER_STORE, ONBOARD_STORE } from '../stores';
+import { recordSubmitStep } from './internals/analytics/record-submit-step';
+import LaunchPad from './internals/steps-repository/launchpad';
+import Processing from './internals/steps-repository/processing-step';
+import {
+	AssertConditionResult,
+	AssertConditionState,
+	Flow,
+	ProvidedDependencies,
+} from './internals/types';
+
+const write: Flow = {
+	name: WRITE_FLOW,
+	get title() {
+		return translate( 'Write' );
+	},
+	useSteps() {
+		useEffect( () => {
+			recordFullStoryEvent( 'calypso_signup_start_free', { flow: this.name } );
```

Comment:
Do we still need to trigger this event? It doesn't seem to align with this new flow.

Reply:
We don't but, in the future, I'm probably going to chat with @nuriapenya about recordings for our Launchpad / onboarding flows.